### PR TITLE
DR: Purpose Statement

### PR DESCRIPTION
**Pull request by noelmace**
record the decision made in https://github.com/blindnet-io/communication-management/issues/86

### CODE REVIEW

File: docs/decision-records/DR-0003-purpose-statement.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+- Good, because some might still consider "Privacy" jargonistic and redundant with "Trust".
+- Bad, because most people still seems to have at least some notion of what it means.
+- Bad, because "Privacy" seems mandatory to allow our purpose statement to sound meaningful enough to developers.
+- Bad, because "Trust" alone doesn't state on which condition the trust is built.
+
+## Links
+
+- [Notions of Privacy](https://github.com/blindnet-io/product-management/blob/main/refs/notion-of-privacy/notion-of-privacy.md)
+
```
Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:118]:
```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```

Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:118]:
```suggestion
- [HBR: The Difference Between Purpose and Mission ](https://hbr.org/sponsored/2016/02/the-difference-between-purpose-and-mission)
```

File: docs/decision-records/DR-0003-purpose-statement.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+| **PR #**   | [32](https://github.com/blindnet-io/openness-framework/pull/32) |
+| **Author** | Noël Macé (noel@blindnet.io)                                    |
+
+## Context and Problem Statement
+
+Before making this decision, blindnet only had a mission statement:
+
+> « Helping developers make software that users love and trust. »
+
+A mission only factually state what an organization is **currently trying** to accomplish.
```
Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:14]:
```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:14]:
```suggestion
A mission only factually states **what** an organization is trying to accomplish.
```
(HBR says "Mission" is about WHAT, and "Purpose is about "WHY")

File: docs/decision-records/DR-0003-purpose-statement.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:16]:
```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:16]:
```suggestion
A purpose statement stays necessary to simply state why the organization exist as a collective, and what drives this collective on a daily basis.
```
```suggestion
A purpose statement is necessary to  state **why** the organization exist as a collective, and what drives this collective on a daily basis.
```
HBR says "Mission" is about WHAT, and "Purpose is about "WHY"

File: docs/decision-records/DR-0003-purpose-statement.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+## Decision Drivers
+
+The purpose statement has to:
+
+- state our common values and goals
+- be straightforward
+- avoid any confusing, ambiguous or jargonistic term
+- be short (few words)
+- be easy to remember and say
+
```
Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:29]:
```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

Comment (milstan) [inline docs/decision-records/DR-0003-purpose-statement.md:29]:
```suggestion

The purpose statement is not:
-  slogan
- a vision statement
```

**Review by milstan — APPROVED** [review]
I made few suggestions for clarity around what is a purpose statement.

**Comment by milstan** [comment]
@noelmace I have no idea why my suggestions appear twice. Feel free to just deal with one.

**Comment by noelmace** [comment]
All suggestions from @milstan have been applied. I've just squashed the commits together and fixed a few typos.

This PR is ready to be merged directly by @milstan after @Vuk-BN's review.

**Pull request merged**